mmv1/products/alloydb/Backup.yaml

```diff
@@ -45,6 +45,21 @@ examples:
     ignore_read_extra:
       - 'reconciling'
       - 'update_time'
+    skip_test: true
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'alloydb_backup_basic_test'
```

Comment:
I'm a little curious why we need a separate test/docs versions for each of these. We didn't need it before, and it makes it a bit harder to review.

Reply:
Previously, we only bootstrapped network, in the test examples, data source network, resource address and networking connection are used. Now we bootstrapped the network, address, and networking connection, and only data source in the test examples. I don't want to change the test examples in the doc, so the old test examples only show in doc, and the new test examples are just for tests.

Reply:
Got it, makes sense!